virt/virtual_machines/vm_networking/virt-attaching-vm-to-external-network.adoc

```diff
@@ -0,0 +1,80 @@
+:_content-type: ASSEMBLY
+[id="virt-attaching-vm-to-external-network"]
+= Attaching a virtual machine to an external network
+include::_attributes/common-attributes.adoc[]
+:context: virt-attaching-vm-to-external-network
+
+toc::[]
+
+{VirtProductName} is installed with a default internal `pod` network. If you want to attach VMs to an external network, however, you must create a Linux _network bridge_.
+
+Depending on the type of networks you are using, it may be necessary to update your xref:../../../virt/node_network/virt-updating-node-network-config.adoc#virt-about-nmstate_virt-updating-node-network-config[node networking configuration].
+
+Once a network bridge is created, you can start virtual machines on the external network in the same manner as you start virtual machines attached to internal networks, by using PXE booting, for example.
+
+= About configuring network bridges
+
+To attach virtual machines to additional networks, you must configure a Linux bridge.
+
+To attach a virtual machine to an additional network:
+
+. Configure a Linux network bridge.
```

Comment:
I think we agreed to move this high-level procedure to the introduction because it covers more than configuring network bridges.

Reply:
I removed the heading 'About' so we have a long intro but it's also comprehensive.